api/v3/Order.php

```diff
@@ -144,7 +144,7 @@ function civicrm_api3_order_create(array $params): array {
   }
 
   $contribution = civicrm_api3('Contribution', 'create', $contributionParams);
-  $contribution['values'][$contribution['id']]['line_item'] = $order->getLineItems();
+  $contribution['values'][$contribution['id']]['line_item'] = array_values($order->getLineItems());
```

Comment:
order api has some funky internal indexing - this sequentials it

- the line_item array only started getting returned in 5.39 - I'm checking the format there

Reply:
OK - this changes it to be consistent with 5.39 - we should merge before the rc is cut